TYPO3.Fluid/Classes/TYPO3/Fluid/Core/Parser/SyntaxTree/ObjectAccessorNode.php

```diff
@@ -88,10 +88,22 @@ public static function getPropertyPath($subject, $propertyPath, RenderingContext
     {
         $propertyPathSegments = explode('.', $propertyPath);
         foreach ($propertyPathSegments as $pathSegment) {
+            $isHasDirectAccess = false;
             try {
-                $subject = ObjectAccess::getProperty($subject, $pathSegment);
+                if (preg_match('/^(is|has)([A-Z].*)/', $pathSegment, $matches) > 0) {
+                    $subject = ObjectAccess::getProperty($subject, lcfirst($matches[2]));
```

Comment:
The readability can be increased by changing the ``pathSegment`` instead of repeating the whole ``try/catch`` block (maybe also rename "pathSegment" in this case):
```php
foreach ($propertyPathSegments as $propertyName) {
  if (preg_match('/^(is|has)([A-Z].*)/', $propertyName, $matches) === 1) {
    $propertyName = lcfirst($matches[2]);
  }
  try {
    $subject = ObjectAccess::getProperty($subject, $propertyName);
  } catch (PropertyNotAccessibleException $exception) {
    $subject = null;
  }
}
```